packages/core/core/src/configDefaults.ts

```diff
@@ -4,6 +4,22 @@ import { join } from "path";
 
 const constants = readJsonSync(join(__dirname, '../constants.json'));
 
+export const embarkConfigDefaults = {
```

Comment:
This is very similar to something in open PR #2260.

See: https://github.com/embarklabs/embark/pull/2260/files#diff-e037f692115931488273a74de0aedc2bR1-R18.

Should the default config's one true home be in `embark-core/constants`?